### DISCUSSION TITLE
Add a piece of documentation to i3 user's guide - floating layout.

### DISCUSSION THREAD

**Discussion by RayZ0rr** [i3/i3#4819] [Ideas]
This might seem opposite to the core purpose of tiling window managers. But say, someone want to have a specific workspace full of floating windows and remaining, say 8 workspaces in tiling mode. This will be helpful for them. Many other tilers also have floating layout apart from other layouts.

In [6.3. Manipulating layout](https://i3wm.org/docs/userguide.html#manipulating_layout) section, add a note that **other possible layouts are possible using [command_criteria] and [commands]** and give example like this: 
```sh
# Floating layout. Float all windows in a workspace.
bindsym $mod+Ctrl+space [workspace=__focused__] floating toggle
```

**Comment by Airblader** [comment]
There's nothing about this that isn't documented; it's just a specific use case, and one that as a tiling window manager we don't encourage to begin with. So I see no value in adding this to the user guide.

It also really isn't a "floating layout", because new windows would still not float.

**Reply by RayZ0rr** [reply]
I see. Can any other `<criteria>` be added to make new windows on that workspace floating as well?

**Reply by Airblader** [reply]
```
for_window [workspace="…"] floating enable
```

But even that doesn't work when moving a tiled window into that workspace. To make it absolutely bullet-proof, you'll need to write a script that monitors the workspace.

**Reply by RayZ0rr** [reply]
if I understand correctly this goes in i3 config file? I was looking for something to bind to a shortcut so that "floating" layout can be toggled on the fly. That's what my command tries to achieve.
```sh
# Floating layout. Float all windows in a workspace.
bindsym $mod+Ctrl+space [workspace=__focused__] floating toggle
```

**Reply by Airblader** [reply]
> I was looking for something to bind to a shortcut so that "floating" layout can be toggled on the fly

You'll need a script to achieve that.
